bin/preupg

```diff
@@ -14,19 +15,18 @@ def main():
     cli = CLI()
     conf = Conf(cli.opts, settings, cli)
     app = Application(conf)
-    ret = 0
-    if cli.opts.debug:
-        ret = app.run()
-    else:
-        try:
-            ret = app.run()
-            logger_debug.debug("Return code is '%s'", ret)
-        except KeyboardInterrupt:
-            print ('\nAssessment interrupted.')
-            return 1
-        except Exception as ex:
-            cli.parser.error(str(ex))
-    return ret
+    try:
+        return_code = app.run()
+        logger_debug.debug("preupg returned '%s'" % return_code)
+        return return_code
+    except KeyboardInterrupt:
+        return settings.ReturnValues.USER_ABORT
+    except Exception as ex:
```

Comment:
this line is not covered by patch for python-2.4. It seems for now that it is the only one problem, just according to usual run of "preupg". I will look at it closer yet today.